### DISCUSSION TITLE
glava --desktop possible?

### DISCUSSION THREAD

**Discussion by se7en-x230** [prasanthrangan/hyprdots#934] [General]
Somebody figured out how to run glava with the --desktop option?
or cava ... any desktop background music visualizer

**Comment by kRHYME7** [comment]
I think it's possible if glava supports Wayland. 
https://github.com/jarcode-foss/glava. 

There is a plugin for this on hyprland, ``` hyprwinwrap ``` but dunno yet how to set it up. You can try

**Comment by kRHYME7** [comment]
@se7en-x230 
Like this ?
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/8d29416c-d9df-4163-8ea7-47743618db3d)


https://github.com/prasanthrangan/hyprdots/assets/53417443/5735b8b5-ec17-429f-a8cd-b5ea5ded5166

**Comment by se7en-x230** [comment]
Wow you managed?

How?

**Reply by kRHYME7** [reply]
https://github.com/hyprwm/hyprland-plugins/tree/main/hyprwinwrap

**Comment by se7en-x230** [comment]
Happy for  now :)


https://github.com/prasanthrangan/hyprdots/assets/69777713/4130a855-16ec-450b-9395-0e072bef3cd1

**Reply by akashgagda** [reply]
Hey bro how you can use Glava can you give details

**Reply by squirticulous** [reply]
how exactly are you doin this
